dashboard/test/ui/features/teacher_tools/level_types/bubble_choice.feature

```diff
@@ -99,14 +101,14 @@ Feature: BubbleChoice
 
```

Comment:
I think you will need to skip `Navigating between a Lab2 sublevel and another Lab2 level` on Safari too

Reply:
Sorry, I'd only looked to the one log file and not the other. Updated so both are off!